README.md

```diff
@@ -52,6 +52,12 @@ We recommend the use of [React Storybook](https://github.com/storybooks/react-st
      $ export CARBON_USE_EXPERIMENTAL_FEATURES=true
      ```
 
+   - `true` to `CARBON_USE_BREAKING_CHANGES` environment variable to test some of the breaking changes for the next release:
```

Comment:
Should we add these notes to a `docs/experimental.md` - `## Feature flags` section instead of placing in our `README.md`?

Reply:
Makes sense for `CARBON_USE_EXPERIMENTAL_FEATURES`. Removed that reference here and created: #1794